### ISSUE TITLE
typst_asset_name not matching latest Typst releases

### ISSUE THREAD

**Issue by mintyfrankie** [yusancky/setup-typst#8]
**Description**
When running the Github action with flag `runs-on: ubuntu-latest`, the workflow returns error `Error: Asset with name typst-x86_64-unknown-linux-gnu.tar.gz not found!`

**How to reproduce**
With the code below:

```
name: Compile PDFs

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout
      uses: actions/checkout@v3

    - name: Setup Typst
      uses: yusancky/setup-typst@v1
      id: setup-typst
      with:
        version: 'v0.5.0'
    - run: typst compile cv.typ
    - run: typst compile letter.typ
```
**Potential reason**
Recent Typst releases have renamed the asset file from `typst-x86_64-unknown-linux-gnu.tar.gz` to `typst-x86_64-unknown-linux-musl.tar.xz`

**Comment by yusancky** [comment]
Thank you very much for raising this issue!

----

I have addressed this aspect in #9.

> **Note**
> However, due to some problems, I am not able to adapt older versions of Typst (i.e. versions before `v0.3.0`) at the moment.

So, if you need to use `v0.5.0`, you only need to change the  action version number.
```diff
- uses: yusancky/setup-typst@v1
+ uses: yusancky/setup-typst@v2
```

**Comment by mintyfrankie** [comment]
Hi @yusancky thanks for the shift fixing!

I will open a quick PR very soon for another nice-to-have feature, it'd be great if you can take a look :)
